src/test/java/com/snack/news/service/NewsServiceTest.java

```diff
@@ -27,44 +30,50 @@
 	private NewsRepository newsRepository;
 
 	@Test
-	public void 뉴스_리스트를_조회할_수_있다() {
+	@DisplayName("뉴스 리스트를 조회할 수 있다")
+	void getNewsListTest() {
 		NewsDto newsDtoWithValidDates = mockNewsDto;
 		when(newsRepository.findByNewsDto(newsDtoWithValidDates)).thenReturn(any());
 
 		newsService.getNewsList(newsDtoWithValidDates);
+		verify(newsRepository).findByNewsDto(any(NewsDto.class));
 	}
 
-	@Test(expected = NewsBadRequestException.class)
-	public void 뉴스_리스트를_조회할_때_날짜가_없다면_예외가_발생한다() {
+	@Test
+	@DisplayName("뉴스 리스트를 조회할 때 날짜가 없다면 예외가 발생한다")
+	void getNewsListTestWhenNoneDate() {
 		NewsDto newsDtoWithNoDates = NewsDto.builder()
 				.title(TEST_TITLE)
 				.content(TEST_CONTENT)
 				.build();
 
-		newsService.getNewsList(newsDtoWithNoDates);
+		assertThrows(NewsBadRequestException.class, () -> newsService.getNewsList(newsDtoWithNoDates));
 	}
 
-	@Test(expected = NewsBadRequestException.class)
-	public void 뉴스_리스트를_조회할_때_날짜가_부적절하다면_예외가_발생한다() {
+	@Test
+	@DisplayName("뉴스 리스트를 조회할 때 날짜가 부적절하다면 예외가 발생한다")
+	void getNewsListTestWhenInvalidDate() {
 		NewsDto newsDtoWithInvalidDates = NewsDto.builder()
 				.title(TEST_TITLE)
 				.content(TEST_CONTENT)
 				.startDateTime(INVALID_START_DATE)
 				.endDateTime(INVALID_END_DATE)
 				.build();
 
-		newsService.getNewsList(newsDtoWithInvalidDates);
+		assertThrows(NewsBadRequestException.class, () -> newsService.getNewsList(newsDtoWithInvalidDates));
 	}
 
 	@Test
-	public void 뉴스를_조회할_수_있다() {
+	@DisplayName("뉴스를 조회할 수 있다")
+	void getNewsTest() {
 		when(newsRepository.findById(anyLong())).thenReturn(Optional.of(mockNews));
 		newsService.getNews(TEST_SOME_ID_LONG);
 	}
 
-	@Test(expected = NewsNotFoundException.class)
-	public void 뉴스를_조회시_ID가_부적절하면_예외가_발생한다() {
+	@Test
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/snack-news/Snack-BE/pullRequest?prid=4863700)